web-ui/src/main/resources/catalog/components/viewer/wmsimport/partials/kmlimport.html

```diff
@@ -14,7 +14,7 @@
 
       <!--KMZ upload-->
       <div class="fu">
-        <button type="button" class="btn btn-info">
+        <button type="button" class="btn btn-default">
```

Comment:
Not sure if related to this change, but in Chrome if I hover in the button depending if hovering at the bottom of the button or in other position the button color is changed to grey or not.

Reply:
That is probably not related to this change. It seems that a transparent layer/div is on top of this button blocking the hover.

This change is just about the colour, the `btn-info` style has not enough colour contrast